src/v2/wallet.js

```diff
@@ -1135,6 +1136,28 @@ Wallet.prototype.prebuildAndSignTransaction = function(params, callback) {
   }).call(this).asCallback(callback);
 };
 
+Wallet.prototype.bumpTransaction = function(params, callback) {
+  return co(function *() {
+    // TODO(BG-9349): change the last check to > 0 and the error message once platform allows multiple transactions to
+    //                be bumped in the same CPFP transaction
+    if (_.isUndefined(params.cpfpTxIds) || !Array.isArray(params.cpfpTxIds) || params.cpfpTxIds.length !== 1) {
+      const error = new Error('expecting cpfpTxIds to be an array of length 1');
+      error.code = 'cpfptxids_not_array';
+      throw error;
+    }
+
+    if (_.isUndefined(params.cpfpFeeRate) && _.isUndefined(params.maxFeeRate)) {
+      debug('neither cpfpFeeRate nor maxFeeRate set in bumpTransaction');
+    }
+
+    if (_.isUndefined(params.maxFee)) {
+      debug('maxFee not set in bumpTransaction');
+    }
+
+    return yield this.prebuildAndSignTransaction(params);
```

Comment:
What are you doing with the tx after it's signed?

Reply:
The result can be used as input to the `submitTransaction` function as it would be for a normal `prebuildAndSignTransaction` call.

Reply:
Why not just send instead of requiring the caller to pass the tx back in submit transaction? This is a behavioral difference from the other specialized tx functions in wallet.js, such as `consolidateUnspents`,  `fanoutUnspents` and `sweep`, which all submit the tx directly. Or maybe consider adding a "nosubmit" param?

Reply:
Added submit to the accelerate workflow. Did not add a "nosubmit" param.